pkg/local_object_storage/metabase/version.go

```diff
@@ -71,3 +80,26 @@ func getVersion(tx *bbolt.Tx) uint64 {
 
 	return 0
 }
+
+var migrateFrom = map[uint64]func(*DB, *bbolt.Tx) error{
+	2: migrateFrom2Version,
+}
+
+func migrateFrom2Version(db *DB, tx *bbolt.Tx) error {
+	tsExpiration := db.epochState.CurrentEpoch() + objectconfig.DefaultTombstoneLifetime
+	bkt := tx.Bucket(graveyardBucketName)
+	c := bkt.Cursor()
+
+	for k, v := c.First(); k != nil; k, v = c.Next() {
+		newVal := make([]byte, addressKeySize, addressKeySize+8)
+		copy(newVal, v)
```

Comment:
imo we should assert `len(v) == addressKeySize`

Reply:
added but i dont know, it is kinda we need a check in every code that takes _any_ value from db that was _not created by this exact code_ (if fact any start except a start from scratch)
returning an error now, not sure what else can be done here

Reply:
> returning an error now, not sure what else can be done here

some error is at least better than silence. When we expect strict value format, and it's unintentionally violated in fact, app should react on its own - remove the shard from the game or completely shutdown. Otherwise, further runtime may be unsafe in relation to stored data